Makefile

```diff
@@ -1,11 +1,11 @@
-REBAR3=$(shell which rebar3 || echo ./rebar3)
+REBAR3=./rebar3
 
 .PHONY: all clean clean_plt compile test doc dialyzer xref ci
 
 EXOMETER_PLT=exometer_core.plt
 DIALYZER_OPTS = # -Wunderspecs
 DIALYZER_APPS = erts kernel stdlib compiler syntax_tools \
-		test_server common_test folsom \
```

Comment:
Why did you remove `test_server`?

Reply:
That has only happened in 19.0, however we still support 18.3 so we should keep that.